x/fibre/spec.md

```diff
@@ -0,0 +1,209 @@
+# Fibre Module Specification
+
+## Abstract
+
+The `x/fibre` module enables validators in the active set to register and manage their fibre service provider information.
```

Comment:
> in the active set

Why only the active set? It likely doesn't happen often but an active validator can be replaced by a bonded validator. If the bonded validator can't register their fibre service provider information ahead of time then they won't service Fibre requests when they become active and there will be some downtime before they register.

Reply:
yeah good point this seems like a premature optimization for state growth 

we can just get rid of that requirement and the deletion mechanism for now in all likely hood